lib/engine/game/g_1849/game.rb

```diff
@@ -436,6 +466,12 @@ def close_corporation(corporation, quiet: false)
           end
           corporation.next_to_par = true if @corporations[index - 1].floated?
           update_garibaldi
+          # code below is for variant rules
+          @log << 'E-token returned to bank' if corporation.e_token
+          corporation.e_token = false if electric_dreams?
+
+          repaid_bond = corporation.loans.pop if bonds? && corporation.loans.any?
+          @loans << repaid_bond if repaid_bond
```

Comment:
Why keep track of loans in the game class? Each corp gets one bond, so they don't need to be tracked except within the corporation.

Reply:
My comment in regards to why do you bother subtracting and adding loans. All you need to do is define total_loans = 1 to accomplish what the linked logic requires. Once the rest of this is ready, I'll add a commit that changes the view code to not require total_loans at all.

Reply:
Alright, I've set NUM_LOANS = 1 and removed all instances of subtracting and adding loans to @loans.